### ISSUE TITLE
Enable screen lock after X second

### ISSUE THREAD

**Issue by zhaoshenhao** [tomalabaster/flutter_app_lock#7]
This could be 2 new features, or can be consolidated into 1. Appreciate if you can add them.

1. If the app is in foregroud, lock screen will be enabled after x second no-op
2. Enable lock screen ONLY after x second when switching apps. This is useful when user is copying data from flutter app to other app and switches back in short duration.

So we can pass 2 parameters to AppLock

noOpAutoLockDuration: 15
switchAppAutoLockDuration: 15

Or as 1 feature:

autoLockDuration: 15

Thanks a lot,

**Comment by tomalabaster** [comment]
Thanks for the suggestions @zhaoshenhao! I'll see what I can do🙂

**Comment by tomalabaster** [comment]
I've

**Comment by tomalabaster** [comment]
@zhaoshenhao I've added in a new property called `backgroundLockLatency`. It's a `Duration` which allows you to control how much time the app is allowed in the background for before showing the lock screen on return.

As for your second feature suggestion, I've opened another issue for tracking that separately: #10.

It has several challenges including measuring interactivity from the keyboard and media such as videos or using the camera. Touch is no problem as a listener could be used but if the timer for this is set low e.g. 30s and the app has a text field for example it would be very frustrating if it kept locking.

Thanks for the suggestions and for using this package! :)
